examples_tests/39.DenoiserTonemapper/exampleInputArguments.txt

```diff
@@ -1,10 +1,10 @@
--COLOR_FILE=../../media/OpenEXR/splitted/color_56_render_0_2_32.exr -CAMERA_TRANSFORM=0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -MEDIAN_FILTER_RADIUS=1 -DENOISER_EXPOSURE_BIAS=10.2 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=0.4,0.8 -OUTPUT=56_render_0_2_32.exr -ALBEDO_FILE=../../media/OpenEXR/splitted/albedo_56_render_0_2_32.exr -NORMAL_FILE=../../media/OpenEXR/splitted/normal_56_render_0_2_32.exr -COLOR_CHANNEL_NAME=color -ALBEDO_CHANNEL_NAME=albedo -NORMAL_CHANNEL_NAME=normal -BLOOM_PSF_FILE=none
--COLOR_FILE=../../media/OpenEXR/splitted/color_56_render_0_2_64.exr -CAMERA_TRANSFORM=0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -MEDIAN_FILTER_RADIUS=1 -DENOISER_EXPOSURE_BIAS=10.2 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=0.4,0.8 -OUTPUT=56_render_0_2_64.exr -ALBEDO_FILE=../../media/OpenEXR/splitted/albedo_56_render_0_2_64.exr -NORMAL_FILE=../../media/OpenEXR/splitted/normal_56_render_0_2_64.exr -COLOR_CHANNEL_NAME=color -ALBEDO_CHANNEL_NAME=albedo -NORMAL_CHANNEL_NAME=normal -BLOOM_PSF_FILE=none
--COLOR_FILE=../../media/OpenEXR/splitted/color_56_render_0_2_128.exr -CAMERA_TRANSFORM=0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -MEDIAN_FILTER_RADIUS=1 -DENOISER_EXPOSURE_BIAS=10.2 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=0.4,0.8 -OUTPUT=56_render_0_2_128.exr -ALBEDO_FILE=../../media/OpenEXR/splitted/albedo_56_render_0_2_128.exr -NORMAL_FILE=../../media/OpenEXR/splitted/normal_56_render_0_2_128.exr -COLOR_CHANNEL_NAME=color -ALBEDO_CHANNEL_NAME=albedo -NORMAL_CHANNEL_NAME=normal -BLOOM_PSF_FILE=none
--COLOR_FILE=../../media/OpenEXR/splitted/color_56_render_0_2_256.exr -CAMERA_TRANSFORM=0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -MEDIAN_FILTER_RADIUS=1 -DENOISER_EXPOSURE_BIAS=10.2 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=0.4,0.8 -OUTPUT=56_render_0_2_256.exr -ALBEDO_FILE=../../media/OpenEXR/splitted/albedo_56_render_0_2_256.exr -NORMAL_FILE=../../media/OpenEXR/splitted/normal_56_render_0_2_256.exr -COLOR_CHANNEL_NAME=color -ALBEDO_CHANNEL_NAME=albedo -NORMAL_CHANNEL_NAME=normal -BLOOM_PSF_FILE=none
\ No newline at end of file
+-COLOR_FILE=../../media/OpenEXR/splitted/color_daily_pt_8.exr -CAMERA_TRANSFORM=1,0,0,0,0.83205,-0.5547,0,0.5547,0.83205 -MEDIAN_FILTER_RADIUS=1 -DENOISER_EXPOSURE_BIAS=10.2 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=0.4,0.8 -OUTPUT=out8_can.exr -ALBEDO_FILE=../../media/OpenEXR/splitted/albedo_daily_pt_8.exr -NORMAL_FILE=../../media/OpenEXR/splitted/normal_daily_pt_8.exr -COLOR_CHANNEL_NAME=color -ALBEDO_CHANNEL_NAME=albedo -NORMAL_CHANNEL_NAME=normal
```

Comment:
jesus don't do esposure bias 10.2, but -2.0, like before so we can at least compare outputs to previous

Reply:
oh, I have copied it from my previous version uh

Reply:
done